package.json

```diff
@@ -89,7 +89,7 @@
     "jest": "^26.4.0",
     "lockfile-lint": "^4.3.7",
     "react": "16.x.x",
-    "react-dom": "16.x.x",
+    "react-dom": "^16.14.0",
```

Comment:
Why update to a specific version here and not `react`?

Reply:
I was attempting to fix the warnings created by react testing library

Reply:
@JAdshead do you want to update `react` to match then?

Reply:
@JamesSingleton i will revert